### PR TITLE
Document pv persistence new statefulset

### PR DESCRIPTION
**Pull request by frankyhollywood**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2740,6 +2740,52 @@ kubectl delete statefulset fairspace-new --namespace fairspace-new --cascade=orp
 This will remove the ``StatefulSet``, but keep all pods and volumes intact.
 After running ``helm upgrade --install`` the ``StatefulSet`` will be recreated and the running pod will be replaced with a pod running the new application version.
 
+===== Keep existing volumes on update of StatefulSet
+
+Any changes on the StatefulSet will require a clean deployment, with new volumes. If you want to change the StatefulSet while keeping your existing volumes follow the procedure as described below.
+
+To preserve your volumes get the volume id's:
```
Comment (ewelinajozwik) [inline README.adoc]:
```suggestion
To preserve your volumes get the volume id's (pv-name's):
```

File: README.adoc [hunk 2]
```diff
@@ -2740,6 +2740,52 @@ kubectl delete statefulset fairspace-new --namespace fairspace-new --cascade=orp
...
+
+Now patch the volumes so they are not deleted automatically when the VolumeClaim is removed:
+
+[source,shell]
+----
+$ kubectl patch pv <pv-name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
+$ kubectl get pv # now shows status 'released'
+----
+
+Uninstall the statefulset and remove the VolumeClaims. In released status the volumes cannot bind to a new claim, therefore remove the claim reference:
```
Comment (ewelinajozwik) [inline README.adoc:2762]:
```suggestion
Uninstall the StatefulSet and remove the VolumeClaims. In released status the volumes cannot bind to a new claim, therefore remove the claim reference:
```

File: README.adoc [hunk 3]
```diff
@@ -2740,6 +2740,52 @@ kubectl delete statefulset fairspace-new --namespace fairspace-new --cascade=orp
...
+
+[source,shell]
+----
+$ kubectl edit pv <pv-name>
+----
+In the editor, remove the 'claimref' section.
+
+The final step is to add the previously gathered Volume id's to the statefulset configuration:
```
Comment (ewelinajozwik) [inline README.adoc]:
```suggestion
The final step is to add the previously gathered volume id's to the StatefulSet configuration:
```

**Review by ewelinajozwik — COMMENTED** [review]
Nice and clear description!

File: README.adoc (hunk 3 above)
Comment (frankyhollywood) [reply README.adoc]: Google capitalizes the name 'StatefulSet' (https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/)

**Pull request merged**